lighthouse.js

```diff
@@ -0,0 +1,18 @@
+const config = {
+  urls: [
+    'http://localhost:7080/news/articles/c9rpqy7pmypo',
+    'http://localhost:7080/news/articles/c85pqyj5m2ko',
```

Comment:
Can these be populated dynamically by reading the `data` folder?

Reply:
As Chris suggests here we could do something like `fs.readdir('data/prod/news/')` and then dynamically make the array. However this could be a future enhancement in a follow up issue as I'm conscious this PR is already doing a lot.

Reply:
We should make it a future PR because the same could be said for a11y.js

Reply:
We should run all our scenarios (lighthouse + a11y) as well, this could be apart of that

Reply:
Could you please open an issue for each lighthouse and a11y to use `fs` and read directory to get their file paths and comment them here for context 😄

Reply:
https://github.com/BBC-News/simorgh/issues/944